pkg/toolchains/rpmbuild.bzl

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""toolchain to provide an rpmbuild binary."""
+
+RpmbuildInfo = provider(
+    doc = """Platform inde artifact.""",
+    fields = {
+        "label": "The path to a target I will build",
+        "path": "The path to a pre-built rpmbuild",
+    },
+)
```

Comment:
I remember in the original example you sent me that me that this depended on a number of other tools, like `strip` in the CC toolchain.

Are there plans to add stuff like that here?  Other things that come to mind are:

- Various compressors and archivers (zip, tar, etc)
- GNU Coreutils

This, of course, can be added incrementally.

Reply:
Oh, strip is interesting for rpm. Also rpm2cpio and cpio, for end to end tests.
Can I add those in another pass. It should simply be a matter of extending the auto configuring toolchain to find them, but then I need to come up with rules to wrap their use. That can be a whole debate in itself. I can imaging a rule like 
```
rpm_extract(
   name=x,
   src_rpm=:rpm,
   table_of_contents="name of toc output",
   extract_files = [...]
)
```
But I don't want to design that as an afterthougth.